models/gitm/gitm_to_netcdf.f90

```diff
@@ -202,7 +202,7 @@ program gitm_to_netcdf
 call get_time(delta_time, nsecs, ndays)
 real_time = ndays + (real(nsecs, r8) / 86400.0_r8)
 
-call nc_put_variable(ncid, 'time', 1, real_time)
+call nc_put_variable(ncid, 'time', real_time)
```

Comment:
This now resolves to the '0d' variant, scalars do not need to supply a length.

Reply:
nice